Packs/IronscalesEventCollector/Integrations/IronscalesEventCollector/README.md

```diff
@@ -12,6 +12,19 @@ Use this integration to fetch email security incidents from Ironscales as XSIAM
 | First fetch | False |
 | Trust any certificate (not secure) | False |
 | Use system proxy settings | False |
+| Collect all incidents | False |
+
+## Scopes
+
+partner.all
+company.all
+partner.company.view
+company.view
+
+## Collect All Incidents
```

Comment:
Should this be Incidents? For XSIAM should they be called Alerts or Issues or Events?